### ISSUE TITLE
Left click sticks, as if holding the button

### ISSUE THREAD

**Issue by azat11** [CemraJC/clickfix#29]
The app is a savior, but sometimes it sticks and drags stuff, on single click or double click, as if I'm holding the button.

Latest version of the app on Win 7. Please let me know if additional info is required.

**Comment by chengnuochen** [comment]
I agree. This program is great to avoid double click but has some problems. Here are two issues I found: 
1. When using OneNote, when I want hold left click (mouse down and hold) and select texts, it does not stop when I release the left key (mouse up). The selection will continue to follow my cursor position. 
2. When I am using WMWare, it turn out that I cannot click the memu bar when the program is active. I have to temporary switch it off to click the memu. 
My OS version is Windows 10 21H1, 19044.2006
